blackjack.py

```diff
@@ -80,50 +89,31 @@ def print_entire_hand(self):
             print card
 
     def check_natural_win(self):
```

Comment:
If the player has an odd number of cards, this method won't return a boolean but will instead return `None`, which works out fine because `bool(None) == False` but the method could be confusing.
Right now you have:
```
    def check_natural_win(self):
        if cond1:
            return cond2
```
You could do:
```
    def check_natural_win(self):
        if cond1:
            return cond2
        return False
```
or:

```
    def check_natural_win(self):
        return cond1 and cond2
```

Reply:
Ah! I see your point. 
I think I like the second solution, makes it most clear to be.
I'll add this change